mne/decoding/tests/test_base.py

```diff
@@ -273,7 +275,12 @@ def test_get_coef_multiclass(n_features, n_targets):
     """Test get_coef on multiclass problems."""
     # Check patterns with more than 1 regressor
     X, Y, A = _make_data(n_samples=30000, n_features=n_features, n_targets=n_targets)
-    lm = LinearModel(LinearRegression()).fit(X, Y)
+    lm = LinearModel(LinearRegression())
+    assert not hasattr(lm, "model_")
+    lm.fit(X, Y)
+    # TODO: modifying non-underscored `model` is a sklearn no-no, maybe should be a
+    # metaestimator?
```

Comment:
as you pass an estimator to another one LinearModel is de facto a metaestimator. You could do lm.model_ = clone(lm.model) to keep lm.model untouched.

Reply:
I should update the comment, it's already a MetaEstimator:

https://github.com/mne-tools/mne-python/blob/672bdf4357f815c63dfab91d9c8e257266bceb21/mne/decoding/base.py#L28

I was worried about the backward compat break where previously people might have looked at `self.model` and now they'd have to look at `self.model_`, but it is a bugfix to make that change so I'll push it probably Tues 👍